website/docs/r/workspace_settings.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_workspace_setting"
+description: |-
+  Manages workspace settings.
+---
+
+# tfe_workspace_settings
+
+Manages or reads execution mode and agent pool settings for a workspace. If [tfe_organization_default_settings](organization_default_settings.html) are used, those settings may be read using a combination of the read-only `overwrites` argument and the setting itself.
```

Comment:
💅 We should probably add an example for using `overwrites`.

Reply:
Since it's a read-only attribute, all the examples I can think of feel a little contrived. I don't think you'd be interested in this unless you were also reading the execution_mode but needed to know whether or not a default was imposed. I'll leave it out for now but if a clear example emerges I will add it.

Reply:
Hmm. The description in this paragraph is a little disorienting, mostly just because the whole overwrites scheme is a bit challenging to get a grip on. I might whip up a proposed reword.